### PR TITLE
[stardog] Various improvements

### PR DESCRIPTION
**Pull request by srueg**
#### What this PR does / why we need it:
Various improvements.

#### Checklist

- [x] [DCO](https://github.com/appuio/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR contains starts with chart name e.g. `[chart]`

### CODE REVIEW

File: stardog/values.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ stardog:
   existingLicenseSecret: stardog-license
   # Custom Stardog config (string value)
   config: |
+  javaArgs: -Xmx3g -Xms1g -XX:MaxDirectMemorySize=3g
```
Comment (anothertobi) [inline stardog/values.yaml]: Since `Xmx` is set, we should set the resource requests and limits as well.

File: stardog/files/jmx-exporter.yaml [hunk 2]
```diff
@@ -9,6 +9,10 @@ rules:
     name: stardog_database_$2
     labels:
       database: $1
+  - pattern: Stardog<00=databases, 01=(.+?), name=(.+)><>(Count|Value)
+    name: stardog_database_$2
```
Comment (anothertobi) [inline stardog/files/jmx-exporter.yaml]: Is it possible to reuse the name?

File: stardog/requirements.lock [hunk 3]
```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: zookeeper
   repository: https://charts.bitnami.com
-  version: 2.2.2
-digest: sha256:65da1c2229daa2c7da1465bb79115109c798f7477c6c26e0375de05dcca552d4
-generated: "2019-05-28T09:11:41.445617223+02:00"
+  version: 3.0.3
```
Comment (srueg) [inline stardog/requirements.lock]: Apparently not compatible with the metrics exporter.

File: stardog/files/jmx-exporter.yaml (hunk 2 above)
Comment (srueg) [reply stardog/files/jmx-exporter.yaml]: Yes, it shouldn't collide. There's an open ticket with Stardog to unify the metric names so we can get rid of this.

**Pull request merged**